inc/theme-options.php

```diff
@@ -539,6 +558,17 @@ function odm_get_legal_disclaimer()
     }
 }
 
+function odm_get_post_types_for_category_page()
+{
+    $options = get_option('odm_options');
+    if (isset($options['category_page'])) {
+        $post_type = array_map('trim', explode(',', $options['category_page']));
+        return $post_type;
+    } else {
+        return false;
```

Comment:
I agree with returning the default value. I will update it. However, the default value will return only this, array('news-article', 'topic', 'profile', 'announcement'); 
Because the main category is not enable for map-layer, but the layer category is used. The post of site-update has no category assigned.